src/main/resources/public/peer.js

```diff
@@ -0,0 +1,257 @@
+"use strict"
+
+const wsProtocol = location.protocol === "https" ? "wss" : "ws" // in a perfect world, it's always wss
+const WEBSOCKET_URL = `${wsProtocol}://${location.host}/ws`
+
+const connectNormalButton = document.getElementById("connectNormal")
```

Comment:
What do you think about an array with those Nodes?

```
["connectNormal", "otherPeerId"].forEach(e =>  document.getElementById("e")
elements["connectNormal"]
```

I'm not sure what the best practice is...

Reply:
I think your proposal seems fine. I did it this way!